### PR TITLE
Stage 2

### PR DESCRIPTION
**Pull request by rawisa**

### CODE REVIEW

File: firefox-ios/README.md [hunk 1]
```diff
@@ -80,4 +80,82 @@ All developers must follow the following Swift style to keep the code orginized:
 
 With small * Exception: they use 4-space indentation instead of 2.
 
-Further info can be find at [Firefox-ios contributor guidelines](https://github.com/mozilla/firefox-ios#contributor-guidelines)
\ No newline at end of file
+Further info can be find at [Firefox-ios contributor guidelines](https://github.com/mozilla/firefox-ios#contributor-guidelines)
```
Comment (robi-y) [inline firefox-ios/README.md]: find->found

File: firefox-ios/README.md [hunk 2]
```diff
@@ -80,4 +80,82 @@ All developers must follow the following Swift style to keep the code orginized:
...
+ 
+ [Stephan Leroux](https://github.com/sleroux) and [Brian Nicholson](https://github.com/thebnich), contribute the most to the tests, code tests, UI tests, etc... 
+
+* **Funders**:
+
+ Funders give money to the project to keep it alive.
+ 
+ One of the best money incoming for firefox are the search engines such as Google, who pays firefox to make Google search engine as the default search engine for firefox.
+ 
+* **General stakeholders**:
```
Comment (robi-y) [inline firefox-ios/README.md]: I would call it development team

File: firefox-ios/README.md [hunk 3]
```diff
@@ -80,4 +80,82 @@ All developers must follow the following Swift style to keep the code orginized:
...
+ The following sequence diagram shows the important part of any webbrowser and how it works, how the MVC works in firefox:
+ 
+ ![](firefox-sequence.png)
+
+[Reference](http://blog.quent.in/assets/wp-content/uploads/2012/04/index.png)
+ 
+ 
+## Missing important feature
+  
+  According to [bugzilla](https://bugzilla.mozilla.org). Firefox is missing an important feature which filed at https://bugzilla.mozilla.org/show_bug.cgi?id=1228089.
```
Comment (robi-y) [inline firefox-ios/README.md]: Is it one feature or several features? not clear from the discussion. Can you identify one which relates to modeling?

File: firefox-ios/README.md [hunk 4]
```diff
@@ -80,4 +80,82 @@ All developers must follow the following Swift style to keep the code orginized:
...
+ * Foundation stakeholder: Andrea Wood
+ * Advocacy stakeholder: Jochai Ben-Avie
+ * Legal: Mika Devi
+ * Web dev: Craig Cook
+ * Analytics: Gareth C
+ * L10N: Flod, Pascal Chevrel
+ 
+ [Reference](https://wiki.mozilla.org/Websites/Mozilla.org/Smart_On)
+ 
+ ## UML - Sequence diagram
```
Comment (robi-y) [inline firefox-ios/README.md:152]: Please mention the source for diagrams and tools used for preparing them

**Comment by robi-y** [comment]
Nice work, see inline comments
I don't understand what does the PR you sent them contains?

File: firefox-ios/README.md (hunk 2 above)
Comment (rawisa) [inline firefox-ios/README.md]: Instead of testers to call them development team ?

File: firefox-ios/README.md (hunk 3 above)
Comment (rawisa) [inline firefox-ios/README.md]: Bugzilla has a list of bugs and missing features, I picked 1 missing feature and added it to forked repo. and send them PR.
I asked you before if I can do that and you told me yes.

File: firefox-ios/README.md (hunk 4 above)
Comment (rawisa) [inline firefox-ios/README.md:152]: Beneath the image there is the source for the diagram, and inside the image there is a url to the tool that was used.

**Comment by rawisa** [comment]
The PR i sent them contains a code I made for a missing feature in there browser.

**Comment by robi-y** [comment]
So please add a link to the PR

**Comment by rawisa** [comment]
can you also reply to the inline comments ? so i make the needed changes ?

File: firefox-ios/README.md (hunk 2 above)
Comment (robi-y) [inline firefox-ios/README.md]: Instead of stakeholders

File: firefox-ios/README.md (hunk 4 above)
Comment (robi-y) [inline firefox-ios/README.md:152]: ok

**Comment by robi-y** [comment]
Replied, let me know when to merge

**Comment by rawisa** [comment]
Did the changes, you can merge if everything is good now ;)

**Comment by robi-y** [comment]
Excellent Work,
Just some minor comments:
- Stakeholder View: what about the user of the browser? are there several
  kind of users which can be identified
- What is actually the missing feature (you only links to it)

**Comment by rawisa** [comment]
Done ;)

**Pull request merged**